src/backend/sgx/data.rs

```diff
@@ -172,3 +176,83 @@ pub fn aesm_socket() -> Datum {
         mesg: None,
     }
 }
+
+pub fn intel_crl() -> Datum {
+    let name = "Intel CRL cache".to_string();
+    let mut crl_file =
+        match sgx_cache_dir() {
+            Ok(p) => p,
+            Err(e) => return Datum {
+                name,
+                pass: false,
+                info: Some(e.to_string()),
+                mesg: Some(
+                    "enarx expects the directory `/var/cache/intel-sgx` to exist and be readable"
+                        .into(),
+                ),
+            },
+        };
+    crl_file.push("crls.der");
+
+    if !crl_file.exists() {
+        return Datum {
+            name,
+            pass: false,
+            info: None,
+            mesg: Some(
+                "Run `enarx platform sgx cache-crl` to generate the Intel CRL cache file".into(),
+            ),
+        };
+    }
+
+    let crls =
+        match std::fs::read(crl_file) {
+            Ok(c) => c,
+            Err(e) => return Datum {
+                name,
+                pass: false,
+                info: Some(e.to_string()),
+                mesg: Some(
+                    "Re-run `enarx platform sgx cache-crl` to generate the Intel CRL cache file"
```

Comment:
This should specify that reading the file failed and why

Reply:
The `info` field gets the error message as on other examples with `Datum`, isn't that the correct place? I did add the path for the file in question.